docs/features/lambda_dead_letter_queue.md

```diff
@@ -1,11 +1,9 @@
 ---
 id: dead_letter_queues
 title: Dead Letter Queues
-hide_title: true
+hide_title: false
 ---
 
-# Cumulus Dead Letter Queues
```

Comment:
This involves a title change.    Intentional?

Reply:
Yeah, intentional. The inclusion of "Cumulus" seemed superfluous and inconsistent with other docs. Let me know if that's a problem though